src/main/java/data/Account.java

```diff
@@ -0,0 +1,113 @@
+// Copyright 2019 Google LLC
```

Comment:
Why is this exact same Account class in the code twice?
Only one copy of it should exist

Reply:
Looks like this is actually the case for all of the code -- it's in here twice?

Reply:
Something must have happened when I was changing the file structure that duplicated the files. I will fix that up soon.